fixtures/common.py

```diff
@@ -22,9 +22,25 @@ def user_client(user):
     return client
 
 
+@pytest.fixture
+def admin_client(user):
```

Comment:
`pytest-django` already provides this one: https://github.com/pytest-dev/pytest-django/blob/master/pytest_django/fixtures.py#L269-L276

Reply:
I looked into that but preferred to avoid another 3rd-party dependency since this view only takes a few lines of code, didn't feel it really fit in as an API url either since it's just a download link for admins.

Reply:
👍